reject_adapter.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+
+# This file is part of election-orchestra.
+# Copyright (C) 2013-2016  Agora Voting SL <agora@agoravoting.com>
```

Comment:
if this file was created in 2016, I think the proper thing to do is to put only 2016 here